.astyleignore

```diff
@@ -1,13 +1,14 @@
 ^BUILD
 ^cmsis
-^components/TARGET_PSA/services/attestation/attestation.h
-^components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/tfm_impl
-^components/TARGET_PSA/services/attestation/qcbor
-^components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/crypto_struct_ipc.h
-^components/TARGET_PSA/TARGET_TFM
-^components/TARGET_PSA/TESTS
 ^features/cryptocell
 ^features/FEATURE_BLE
+^features/FEATURE_EXPERIMENTAL_API/TARGET_MBED_PSA_SRV
```

Comment:
I don't think we should ignore the entire `TARGET_MBED_PSA_SRV` folder.

Reply:
The `TARGET_MBED_PSA_SRV` folder was ignored previously, as it was under `features/mbedtls`.

Reply:
OK, I see we have another TARGET_MBED_PSA_SRV folder under TARGET_PSA that is for everything not from Mbed TLS, because we wanted to use PSA Crypto APIs on non-PSA targets.